src/components/LayerUI.tsx

```diff
@@ -111,8 +111,11 @@ const LayerUI = ({
 }: LayerUIProps) => {
   const device = useDevice();
 
-  const childrenComponents =
-    ReactChildrenToObject<UIChildrenComponents>(children);
+  const [childrenComponents, restChildren] =
+    partitionReactChildren<UIChildrenComponents>(children, {
```

Comment:
`splitReactChildren` ?

Reply:
split is related to strings. But we can do just `getReactChildren`.